### ISSUE TITLE
Is this still supported project?

### ISSUE THREAD

**Issue by Ahmed-Ali** [airgap-it/airgap-vault#74]
Hi,
While I find this project very appealing all over, I wonder how serious and long term supported it is?
Also for some reasons it isn't as popular as other software wallets, any reasons why? I mean is there obvious down sides that people aren't considering it?

**Comment by AndreasGassmann** [comment]
Hi, thanks for reaching out.

We are very committed to the project and have no plans to stop working on it anytime soon.

The repositories can sometimes seem a bit stale because the development is happening in our private gitlab. Master is only updated when a new version is about to be released, so there are often large periods of "inactivity", but there is constant development going on internally. This is something we want to change in the near future to make everything more transparent.

As for the popularity, I can't really tell you. We are a very technical team with background in mobile security and blockchain technologies, we never really had a marketing team that pushed the project. At the time when we launched, there was also very limited interest in such a security focussed app, so it never really caught on. But we have now actively started pushing the project on social media. We are planning to regularly release new blog posts on medium and are adding our wallet to as many lists as we can find to increase awareness.

We also have some very exciting developments going on right now that will improve the usability and interoperability of the Vault. For example, in one of the next releases we will add support for PSBTs (Bitcoin), which will make our wallet compatible with more advanced desktop wallets such as Sparrow or Electrum. We are also in talks with MetaMask about an integration into their extension (basically allowing you to use AirGap Vault as a signer in MetaMask, completely air-gapped).

So to sum it up, I don't think there is an obvious down side, it's basically just lack of awareness.

**Comment by Ahmed-Ali** [comment]
That's all sounds good and great!
Given your technical background, you probably know that one of the basic signals when researching a 3rd party dependency, is the community using it. I.e the bigger the community the more likely this is a reliably dependency to use (and the opposite gives the opposite signal). Hence raising awareness around this project and growing the community, in my opinion, is a key to its adoption. As it also needs to be battle tested by more people.

I personally find it covers all my core needs (open source, transparent, anonymous, doesn't require specialised hardware, which help not to draw attention, totally airgaped, etc).

I will give it a go with conservative approach till I feel more comfortable to fully move towards it.

Thanks for the prompt response!

**Comment by AndreasGassmann** [comment]
Thanks for the feedback, I agree with your points.

Let us know what you think. If you are primarily interested in Bitcoin, then you should keep this issue in mind, which is now one of the highest priorities: https://github.com/airgap-it/airgap-vault/issues/73
